docs/root/configuration/overview/v2_overview.rst

```diff
@@ -158,6 +158,11 @@ on 127.0.0.3:5678 is provided below:
       type: STATIC
       lb_policy: ROUND_ROBIN
       http2_protocol_options: {}
+      upstream_connection_options:
+        tcp_keepalive:
+          keepalive_probes: 3
```

Comment:
It feels kind of bad to have boiler plate with magic numbers as part of our set examples. Can we replace this with just:
```
     # tcp_keepalive: configure a TCP keep-alive due to issue XYZ if applicable.
```
or something like this. I grok the problem but don't like the support/maintenance burden of many different boiler plate configs and settings.